src/PersistentList.jl

```diff
@@ -51,21 +51,16 @@ Base.iterate(::AbstractList, l::PersistentList) = (head(l), tail(l))
 Base.isequal(a::AbstractArray, l::PersistentList) = isequal(l, a)
 Base.isequal(l::PersistentList, a::AbstractArray) =
     isequal(length(l), length(a)) && all((el) -> el[1] == el[2], zipd(l, a))
+import Base.==
```

Comment:
good catch!

Reply:
Maybe move this to the beginning of the file, since there are `==` definitions above also. I'm not sure if the import will work if it's after the previous ==s.

Reply:
Oh, I see you removed this; maybe it's not necessary because it's imported by a file that includes this?